mteb/tasks/Retrieval/eng/TopiOCQARetrieval.py

```diff
@@ -0,0 +1,91 @@
+from datasets import load_dataset
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class TopiOCQARetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="TopiOCQA",
+        dataset={
+            "path": "McGill-NLP/TopiOCQA",
+            "revision": "66cd1dbf5577c653ecb99b385200f08e15e12f30",
+        },
+        reference="https://mcgill-nlp.github.io/topiocqa",
+        description=(
+            "TopiOCQA (Human-in-the-loop Attributable Generative Retrieval for Information-seeking Dataset)"
+            "is information-seeking conversational dataset with challenging topic switching phenomena."
+            "It consists of conversation histories along with manually labelled relevant/gold passage."
+        ),
+        type="Retrieval",
+        category="s2p",
+        eval_splits=["validation"],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_10",
+        date=None,
+        form=["written"],
+        domains=["Encyclopaedic"],
+        task_subtypes=["Conversational retrieval"],
+        license="cc-by-nc-sa-4.0",
+        socioeconomic_status=None,  # TODO: Not sure that this refers to
+        annotations_creators="human-annotated",
+        dialect=None,
+        text_creation=None,  # TODO: Not sure what to put here, queries are "created", documents are "found"
```

Comment:
@orionw Do you know what exactly does `text_creation` refer to? In this dataset the queries were constructed by MTurkers and docs are paragraphs from Wikipedia

Reply:
Good question, you can find the categories here: https://github.com/embeddings-benchmark/mteb/blob/main/mteb/abstasks/TaskMetadata.py#L58

Reply:
Lets go with `text_creation="found"` referring to the documents as per our discussion and `socioeconomic_status="mixed"` for Wikipedia documents.